CONTRIBUTORS.md

```diff
@@ -55,6 +55,7 @@ The following have contributed code and/or documentation to this repository.
 - Ian Kaneshiro <ian@ctrliq.com>, <iankane@umich.edu>
 - Jack Morrison <morrisonjc@ornl.gov>, <jack@rescale.com>
 - Jacob Chappell <chappellind@gmail.com>, <jacob.chappell@uky.edu>
+- Jarrett Dixon <jarrettdixon1@gmail.com>
```

Comment:
Please put your Sylabs email address here... you can have your GMail address too if you'd like.

Reply:
updated